tensorboard_plugin_wit/wit_plugin.py

```diff
@@ -86,6 +86,24 @@ def __init__(self, context):
                             'authorized_groups' in context.flags and
                             context.flags.authorized_groups != '')
 
+    self.custom_predict_fn = None
+    if context.flags.custom_predict_fn:
+      try:
+        import importlib.util as iu
+        spec = iu.spec_from_file_location("custom_predict_fn", context.flags.custom_predict_fn)
+        module = iu.module_from_spec(spec)
+        spec.loader.exec_module(module)
+        self.custom_predict_fn = module.custom_predict_fn
+        logger.info("custom_predict_fn loaded.")
+
+      except Exception as e:
+        logger.info("Failed to load the custom predict function.")
```

Comment:
nit: should this be logger.error instead? The default logger level is I think 'warning' so this may not be shown.

Reply:
+1